### PR TITLE
Add pip3 as an entry point

### LINKED ISSUE #51: Create an additional entry point for pip3??
Issue:
A few instructions on the web like to specify `pip3` instead of `pip`, but if you try to call `pip3` you would likely get ubuntu's pip version:

```
12:40 $ conda activate base
(base) ✔ ~ 
12:40 $ which pip3
/usr/bin/pip3
```

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /home/mark/miniconda3:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       0_gnu    conda-forge
anaconda-client           1.7.2                      py_0    conda-forge
attrs                     19.3.0                     py_0    conda-forge
beautifulsoup4            4.9.1            py37hc8dfbb8_0    conda-forge
blinker                   1.4                        py_1    conda-forge
boolean.py                3.7                        py_0    conda-forge
brotlipy                  0.7.0           py37h8f50634_1000    conda-forge
bzip2                     1.0.8                h516909a_2    conda-forge
ca-certificates           2020.4.5.1           hecc5488_0    conda-forge
certifi                   2020.4.5.1       py37hc8dfbb8_0    conda-forge
cffi                      1.14.0           py37hd463f26_0    conda-forge
chardet                   3.0.4           py37hc8dfbb8_1006    conda-forge
clyent                    1.2.2                      py_1    conda-forge
conda                     4.8.3            py37hc8dfbb8_2    ramonaoptics
conda-build               3.19.2           py37hc8dfbb8_2    conda-forge
conda-forge-pinning       2020.05.18.14.31.44               0    conda-forge
conda-package-handling    1.6.0            py37h8f50634_2    conda-forge
conda-smithy              3.7.3                      py_0    conda-forge
cookies                   2.2.1                      py_0    conda-forge
cryptography              2.9.2            py37hb09aad4_0    conda-forge
curl                      7.69.1               h33f0ec9_0    conda-forge
decorator                 4.4.2                      py_0    conda-forge
deprecated                1.2.10             pyh9f0ad1d_0    conda-forge
expat                     2.2.9                he1b5a44_2    conda-forge
filelock                  3.0.10                     py_0    conda-forge
gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
git                       2.26.2          pl526h5e3e691_0    conda-forge
gitdb                     4.0.5                      py_0    conda-forge
gitpython                 3.1.2                      py_0    conda-forge
glob2                     0.7                        py_0    conda-forge
icu                       67.1                 he1b5a44_0    conda-forge
idna                      2.9                        py_1    conda-forge
importlib-metadata        1.6.0            py37hc8dfbb8_0    conda-forge
importlib_metadata        1.6.0                         0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
isodate                   0.6.0                      py_1    conda-forge
jinja2                    2.11.2             pyh9f0ad1d_0    conda-forge
jsonschema                3.2.0            py37hc8dfbb8_1    conda-forge
jupyter_core              4.6.3            py37hc8dfbb8_1    conda-forge
krb5                      1.17.1               h2fd8d38_0    conda-forge
ld_impl_linux-64          2.34                 h53a641e_0    conda-forge
libarchive                3.3.3             h3a8160c_1008    conda-forge
libcurl                   7.69.1               hf7181ac_0    conda-forge
libedit                   3.1.20170329      hf8c457e_1001    conda-forge
libffi                    3.2.1             he1b5a44_1007    conda-forge
libgcc-ng                 9.2.0                h24d8f2e_2    conda-forge
libgomp                   9.2.0                h24d8f2e_2    conda-forge
libiconv                  1.15              h516909a_1006    conda-forge
liblief                   0.9.0                hf8a498c_1    conda-forge
libsolv                   0.7.13               h8b12597_2    conda-forge
libssh2                   1.8.2                h22169c7_2    conda-forge
libstdcxx-ng              9.2.0                hdf63c60_2    conda-forge
libxml2                   2.9.10               h72b56ed_1    conda-forge
license-expression        1.2                        py_0    conda-forge
lz4-c                     1.9.2                he1b5a44_1    conda-forge
lzo                       2.10              h14c3975_1000    conda-forge
mamba                     0.2.9            py37h6d3251c_0    conda-forge
markupsafe                1.1.1            py37h8f50634_1    conda-forge
mock                      4.0.2            py37hc8dfbb8_0    conda-forge
more-itertools            8.3.0                      py_0    conda-forge
msrest                    0.6.10                     py_0    conda-forge
nbformat                  5.0.6                      py_0    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
oauthlib                  3.0.1                      py_0    conda-forge
openssl                   1.1.1g               h516909a_0    conda-forge
packaging                 20.4               pyh9f0ad1d_0    conda-forge
patchelf                  0.10                 he1b5a44_0    conda-forge
pcre                      8.44                 he1b5a44_0    conda-forge
perl                      5.26.2            h516909a_1006    conda-forge
pip                       20.1               pyh9f0ad1d_0    conda-forge
pkginfo                   1.5.0.1                    py_0    conda-forge
pluggy                    0.13.1           py37hc8dfbb8_1    conda-forge
prompt-toolkit            3.0.5                      py_0    conda-forge
prompt_toolkit            3.0.5                         0    conda-forge
psutil                    5.7.0            py37h8f50634_1    conda-forge
py                        1.8.1                      py_0    conda-forge
py-lief                   0.9.0            py37he1b5a44_1    conda-forge
pycosat                   0.6.3           py37h8f50634_1004    conda-forge
pycparser                 2.20                       py_0    conda-forge
pycrypto                  2.6.1           py37h8f50634_1004    conda-forge
pygithub                  1.43.6           py37hc8dfbb8_1    conda-forge
pygments                  2.6.1                      py_0    conda-forge
pyjwt                     1.7.1                      py_0    conda-forge
pyopenssl                 19.1.0                     py_1    conda-forge
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyrsistent                0.16.0           py37h8f50634_0    conda-forge
pysocks                   1.7.1            py37hc8dfbb8_1    conda-forge
pytest                    5.4.2            py37hc8dfbb8_0    conda-forge
python                    3.7.6           h8356626_5_cpython    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
python-libarchive-c       2.9                      py37_0    conda-forge
python_abi                3.7                     1_cp37m    conda-forge
pytz                      2020.1             pyh9f0ad1d_0    conda-forge
pyyaml                    5.3.1            py37h8f50634_0    conda-forge
readline                  8.0                  hf8c457e_0    conda-forge
requests                  2.23.0             pyh8c360ce_2    conda-forge
requests-oauthlib         1.2.0                      py_0    conda-forge
responses                 0.10.14            pyh9f0ad1d_0    conda-forge
ripgrep                   12.1.0               h516909a_0    conda-forge
ruamel.yaml               0.16.6           py37h8f50634_1    conda-forge
ruamel.yaml.clib          0.2.0            py37h8f50634_1    conda-forge
ruamel_yaml               0.15.80         py37h8f50634_1001    conda-forge
scrypt                    0.8.13           py37hb09aad4_3    conda-forge
setproctitle              1.1.10          py37h8f50634_1002    conda-forge
setuptools                46.4.0           py37hc8dfbb8_0    conda-forge
six                       1.14.0                     py_1    conda-forge
smmap                     3.0.4              pyh9f0ad1d_0    conda-forge
soupsieve                 2.0.1            py37hc8dfbb8_0    conda-forge
sqlite                    3.30.1               hcee41ef_0    conda-forge
tk                        8.6.10               hed695b0_0    conda-forge
toolz                     0.10.0                     py_0    conda-forge
tqdm                      4.46.0             pyh9f0ad1d_0    conda-forge
traitlets                 4.3.3            py37hc8dfbb8_1    conda-forge
urllib3                   1.25.9                     py_0    conda-forge
vsts-python-api           0.1.22                     py_0    conda-forge
wcwidth                   0.1.9              pyh9f0ad1d_0    conda-forge
wheel                     0.34.2                     py_1    conda-forge
wrapt                     1.12.1           py37h8f50634_1    conda-forge
xonsh                     0.9.18           py37hc8dfbb8_0    conda-forge
xz                        5.2.5                h516909a_0    conda-forge
yaml                      0.2.4                h516909a_0    conda-forge
zipp                      3.1.0                      py_0    conda-forge
zlib                      1.2.11            h516909a_1006    conda-forge
zstd                      1.4.4                h6597ccf_3    conda-forge


```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : base
    active env location : /home/mark/miniconda3
            shell level : 1
       user config file : /home/mark/.condarc
 populated config files : /home/mark/.condarc
          conda version : 4.8.3
    conda-build version : 3.19.2
         python version : 3.7.6.final.0
       virtual packages : __glibc=2.31
       base environment : /home/mark/miniconda3  (writable)
           channel URLs : 
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/mark/miniconda3/pkgs
                          /home/mark/.conda/pkgs
       envs directories : /home/mark/miniconda3/envs
                          /home/mark/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.3 requests/2.23.0 CPython/3.7.6 Linux/5.4.0-31-generic ubuntu/20.04 glibc/2.31
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False


```
</details>

### PR DESCRIPTION
**Pull request by hmaarrfk**
Fixes #51 
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender


I guess we aren't building this for python 2.7 anymore? I added the selector just in case.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,17 +10,18 @@ source:
...
+  number: 1
   disable_pip: true
   entry_points:
     - pip = pip._internal.cli.main:main
+    - pip3 = pip._internal.cli.main:main
 
 requirements:
   host:
-    - python
+    - python 3
```
Comment (jjhelmus) [inline recipe/meta.yaml:21]:
```suggestion
    - python >=3
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,17 +10,18 @@ source:
...
     - setuptools
   run:
-    - python
+    - python 3
```
Comment (jjhelmus) [inline recipe/meta.yaml:24]:
```suggestion
    - python >=3
```

**Comment by jjhelmus** [comment]
The [knowledge base](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python) recommends using `python >=3` when a package only supports Python 3.  Other than that this looks good. Thanks for putting this together @hmaarrfk

**Comment by hmaarrfk** [comment]
Ah thanks for the review.

Thanks for undertanding why this is useful!

**Pull request merged**